index.js

```diff
@@ -31,6 +31,10 @@ if (process.env.hasOwnProperty('GCLOUD_DEBUG_LOGLEVEL')) {
 if (process.env.hasOwnProperty('GCLOUD_DEBUG_DISABLE')) {
   config.enabled = false;
 }
+if (process.env.hasOwnProperty('GCLOUD_DEBUG_RELATIVE_REPOSITORY_PATH')) {
```

Comment:
Is this config important enough to warrant an config and an environment variable? I don't expect people will change this from run-to-run, so a config option is probably enough.